Samples/GLFWDemo/CMakeLists.txt

```diff
@@ -83,6 +83,12 @@ if(MSVC)
     )
 endif()
 
+if(PLATFORM_FREEBSD)
+    if(EXISTS /usr/local/include)
+        include_directories(/usr/local/include)
+    endif()
+endif()
+
```

Comment:
`Samples/GLFWDemo` does not pull in `Diligent-BuildSettings`

If that can be fixed, I can remove this bit and then I think the merge can happen.